base/markdown/render/plain.jl

```diff
@@ -37,6 +37,46 @@ function plain(io::IO, list::List)
     end
 end
 
+function plain(io::IO, md::Table)
+    col_widths = reduce(max, map(cell -> map(ansi_length, cell), md.rows))
+
+    for (n, row) in enumerate(md.rows)
+        for (i, h) in enumerate(row)
+            # TODO use not terminal version of print_align
+            if i != 1
+                print(io, " | ")
+            end
+            print(io, " " ^ (col_widths[i] - ansi_length(h)))
+            plaininline(io, h)
+        end
+        println(io, "")
+
+        if n == 1
+            for (j, w) in enumerate(col_widths)
+                if j != 1
+                    print(io, " | ")
+                end
+                a = typeof(md.align) == Symbol ? md.align : md.align[j]
+                print(io, _dash(w, a))
+            end
+            println(io, "")
+        end
+    end
+end
+
+function _dash(width, align)
+    if align == :l
+        return ":" * "-" ^ max(1, width - 1)
+    elseif align == :r
+        return "-" ^ max(1, width - 1) * ":"
```

Comment:
I had this in just in case we changed the default. I did wonder if this could (later) be configurable i.e. originally I had a :default symbol but for simplicity just changed to the latex way.